src/unix/async.c

```diff
@@ -61,14 +61,43 @@ int uv_async_send(uv_async_t* handle) {
   if (ACCESS_ONCE(int, handle->pending) != 0)
```

Comment:
Although unrelated, would somebody care to explain the reason for this `ACCESS_ONCE()` call? Thanks

Reply:
cmpxchg is a heavyweight operation compared to a simple load, so this is a fast opt-out.

Reply:
:+1: Thanks for the explanation.